packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/core",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Lets go to 1.6.0. 

Please also update the https://github.com/actions/toolkit/blob/main/packages/core/RELEASES.md file with this change under the version 1.6.0